src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -48,8 +48,15 @@
 
 namespace Dynamo.Models
 {
+    /// <summary>
+    /// Interface provides Engine Controller.
```

Comment:
This class creates an interface for Engine controller. 
Add a cref to Engine Controller.